packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -108,6 +108,9 @@ export default function HeaderEditMode() {
 		};
 	}, [] );
 
+	const { get: getPreference } = useSelect( preferencesStore );
+	const hasFixedToolbar = getPreference( editSiteStore.name, 'fixedToolbar' );
```

Comment:
Should we make this a hook so we can share the code?

Reply:
To me a two liner is not worth a hok. I would be if the two liner would be some arcane expression or something but this is really basic.

Reply:
The same comment as above - https://github.com/WordPress/gutenberg/pull/52123/files#r1252980401.